src/main/java/com/verily/genomewarp/utils/GenomeRangeUtils.java

```diff
@@ -38,20 +42,20 @@
 
```

Comment:
All of these functions that now take as input or return TreeMaps should update their signatures to explicitly request TreeMap, since the iteration order is something that is relied upon for code correctness.